vm-setup/roles/packages_installation/tasks/main.yml

```diff
@@ -4,32 +4,33 @@
       package:
         name: "{{ packages.ubuntu.common.packages}}"
         state: present
-      become: true
     - name: Install packages using standard package manager for Ubuntu 18.04
       when: ansible_distribution_version == "18.04"
       package:
         name: "{{ packages.ubuntu.bionic.packages}}"
         state: present
-      become: true
     - name: Install packages using standard package manager for Ubuntu 20.04
       when: ansible_distribution_version == "20.04"
       package:
         name: "{{ packages.ubuntu.focal.packages}}"
         state: present
-      become: true
     - name: Install packages using pip3
       pip:
         executable: "pip3"
         name: "{{ packages.ubuntu.pip3 }}"
         state: present
-      become: true
```

Comment:
In your other PR I have seen you changing `true` to `yes`, should we be consistent?

Reply:
Sure, will fix it. Thanks.